desktop/apps/collect_art/templates/meta.jade

```diff
@@ -1,13 +1,13 @@
-title Buy Art | Artsy
+title Collect Art | Artsy
 - var description = 'An art collection for every budget'
-meta( property='og:title', content='Buy Art | Artsy' )
+meta( property='og:title', content='Collect Art | Artsy' )
 meta( name='description', content= description )
 meta( property='og:description', content= description )
 meta( property='twitter:description', content= description )
-link( rel='canonical', href='#{sd.APP_URL}/buy-art' )
-meta( property='og:url', content='#{sd.APP_URL}/buy-art' )
+link( rel='canonical', href='#{sd.APP_URL}/collect-art' )
+meta( property='og:url', content='#{sd.APP_URL}/collect-art' )
 //- meta( property='og:image', content= asset('/images/og_image.jpg') )
 meta( property='og:type', content='website' )
 meta( property='twitter:card', content='summary' )
 meta( name='viewport', content="width=device-width, initial-scale=1, shrink-to-fit=yes" )
-link( type='text/css', rel='stylesheet', href=asset('/assets/buy_art.css') )
+link( type='text/css', rel='stylesheet', href=asset('/assets/collect_art.css') )
```

Comment:
@mzikherman Thanks Matt for letting us know. I wasn't aware of the existence of the `blank` page template. If it's a consistent pattern, then we should do that before merging.